.env.spadev0

```diff
@@ -0,0 +1,13 @@
+LOTUS_API_URL=https://api.node.glif.io/rpc/v1
+LOTUS_API_TOKEN=
+QUEUE_MONGO_URI=mongodb+srv://user:pass@host/?retryWrites=true&w=majority
+QUEUE_MONGO_DATABASE=test
+RESULT_MONGO_URI=mongodb+srv://user:pass@host/?retryWrites=true&w=majority
+RESULT_MONGO_DATABASE=test
+STATEMARKETDEALS_MONGO_URI=mongodb+srv://user:pass@host/?retryWrites=true&w=majority
+STATEMARKETDEALS_MONGO_DATABASE=test
```

Comment:
I think we can remove these env vars as they aren't needed in the spade integration